packages/skeleton/src/lib/components/InputChip/InputChip.svelte

```diff
@@ -223,21 +219,24 @@
 		chipValues = chipValues;
 	}
 
-	function addChipInternally(event: SvelteEvent<SubmitEvent, HTMLFormElement>): void {
-		event.preventDefault();
-		// Validate
-		inputValid = validate();
-		// When the onInvalid hook is present
-		if (inputValid === false) {
-			/** @event {{ event: Event, input: any  }} invalid - Fires when the input value is invalid. */
-			dispatch('invalid', { event, input });
-			return;
+	function onKeyUpHandler(event: KeyboardEvent): void {
+		if (event.key === 'Enter') {
```

Comment:
Bit of a nitpick, but I would just use and early return: `if (event.key !== 'Enter') return`